backend/channels.py

```diff
@@ -682,50 +1022,114 @@ async def queue_background_channel_update_tasks(config):
     # Update TVH
     from backend.api.tasks import TaskQueueBroker
     task_broker = await TaskQueueBroker.get_instance()
-    # Configure TVH with the list of channels
-    await task_broker.add_task({
-        'name':     'Configuring TVH channels',
-        'function': publish_bulk_channels_to_tvh_and_m3u,
-        'args':     [config],
-    }, priority=11)
-    # Configure TVH with muxes
-    await task_broker.add_task({
-        'name':     'Configuring TVH muxes',
-        'function': publish_channel_muxes,
-        'args':     [config],
-    }, priority=12)
-    # Map all services
-    await task_broker.add_task({
-        'name':     'Mapping all TVH services',
-        'function': map_all_services,
-        'args':     [config],
-    }, priority=13)
-    # Clear out old channels
-    await task_broker.add_task({
-        'name':     'Cleanup old TVH channels',
-        'function': cleanup_old_channels,
-        'args':     [config],
-    }, priority=14)
-    # Fetch additional EPG data from the internet
-    from backend.epgs import update_channel_epg_with_online_data
+    
+    current_tasks = await task_broker.get_pending_tasks()
```

Comment:
There is no need to add `if 'Configuring TVH channels' not in task_names:` in these as the TaskQueueBroker.add_task method already has this check built-in
```
    async def add_task(self, task, priority=100):
        if task['name'] in self.__task_names:
            self.__logger.debug("Task already queued. Ignoring.")
            return
```